models/intermediate/social_media_reporting__instagram_posts_reporting.sql

```diff
@@ -18,7 +18,12 @@ with report as (
         'instagram' as platform,
         coalesce(sum(comment_count),0) as comments,
         coalesce(sum(like_count),0) as likes,
-        sum(coalesce(carousel_album_impressions,0) + coalesce(story_impressions,0) + coalesce(video_photo_impressions, 0)) as impressions
+        sum(
+            coalesce(carousel_album_views, carousel_album_impressions, 0) 
+            + coalesce(story_views, story_impressions, 0) 
+            + coalesce(video_photo_views, video_photo_impressions, 0) 
+            + coalesce(reel_views, 0)
```

Comment:
I think I agree on prioritizing the new fields in the coalesces, but prioritizing the old ones would be less of a breaking change (though perhaps less consistent overall). 

Is this what we've typically done in other new field replacement/deprecation rollouts?

Reply:
I thought we usually prioritize what will be the norm going forward. There should never be both populated for a particular record, so I think the order isn't too crucial in this case.